StripeUICore/StripeUICore/Source/Elements/ElementsUI.swift

```diff
@@ -95,6 +102,7 @@ import UIKit
         public var primary = UIColor.systemBlue
         public var parentBackground = UIColor.systemBackground
         public var background = ElementsUI.backgroundColor
+        public var disabledBackground = ElementsUI.disabledBackgroundColor
```

Comment:
might want to check with Elements folks on whether there are any standards around which colors/options we give - am not sure; ex. @yuki-stripe

Reply:
This is value is copied from [InputFormColors.disabledBackgroundColor](https://github.com/stripe/stripe-ios/blob/1b09a455d04b2fa889fe3f8c8ed680bdbdc15a3d/StripeUICore/StripeUICore/Source/Helpers/InputFormColors.swift#L39 )